apps/console/src/features/identity-providers/models/endpoints.ts

```diff
@@ -25,4 +25,6 @@ export interface IDPResourceEndpointsInterface {
     identityProviders: string;
     localAuthenticators: string;
     multiFactorAuthenticators: string;
+    notificationSenders: string;
+
```

Comment:
Shall we remove the additional newline?

Reply:
resolved in https://github.com/wso2/identity-apps/pull/3699/commits/89a5b7ce383f598b61ffaab02e48b0a23c2bf27a